test/integration/commands/apply/index.test.ts

```diff
@@ -1,52 +1,65 @@
 import { expect } from '@oclif/test'
-import { Environment, Space, createClient } from 'contentful-management'
+import { Space, createClient } from 'contentful-management'
 import fs from 'fs'
-import { ApplyTestContext, TestContext, createEnvironment } from './../bootstrap'
+import { ApplyTestContext, TestContext, createEnvironments } from './../bootstrap'
 import fancy from './../register-plugins'
-import path from 'path'
 
-const spaceId = process.env.CONTENTFUL_SPACE_ID!
-if (!spaceId) {
-  throw new Error('Please provide a `CONTENTFUL_SPACE_ID`')
-}
+describe('Apply command flow', () => {
+  const spaceId = process.env.CONTENTFUL_SPACE_ID!
+  if (!spaceId) {
+    throw new Error('Please provide a `CONTENTFUL_SPACE_ID`')
+  }
 
-const cmaToken = process.env.CONTENTFUL_INTEGRATION_TEST_CMA_TOKEN!
-if (!cmaToken) {
-  throw new Error('Please provide a `CONTENTFUL_INTEGRATION_TEST_CMA_TOKEN`')
-}
+  const cmaToken = process.env.CONTENTFUL_INTEGRATION_TEST_CMA_TOKEN!
+  if (!cmaToken) {
+    throw new Error('Please provide a `CONTENTFUL_INTEGRATION_TEST_CMA_TOKEN`')
+  }
 
-// Should be dynamically created later.
-const changesetFilePath = path.join(__dirname, './changesets/1-add-entry.json')
-console.log({ changesetFilePath })
-let testContext: ApplyTestContext
-let testSpace: Space
+  const changesetFilePath = './changeset.json'
+  let applyTestContext: ApplyTestContext
+  let testSpace: Space
+  let createTestContext: TestContext
 
-before(async () => {
-  const client = createClient({ accessToken: cmaToken })
-  testSpace = await client.getSpace(spaceId)
-  const { environmentId, teardown } = await createEnvironment(testSpace)
-  testContext = {
-    spaceId: testSpace.sys.id,
-    environmentId: environmentId,
-    changesetFilePath,
-    cmaToken,
-    teardown,
-  }
-})
+  before(async () => {
+    const client = createClient({ accessToken: cmaToken })
+    testSpace = await client.getSpace(spaceId)
+    const environmentsContext = await createEnvironments(testSpace)
+    if (!environmentsContext) {
+      throw new Error('Environments were not created successfully')
```

Comment:
why do we need this, isn't an error thrown anyways if something goes wrong with the `testUtils` (https://github.com/contentful/contentful-merge/blob/feat%2Fintegration-tests-part-2/test/integration/commands/bootstrap.ts#L68

Reply:
I double-check and the reason of that check is typescript, `createEnvironments` returns `undefined` or result